conf/prod.routes

```diff
@@ -1,5 +1,5 @@
 # Add all the application routes to the app.routes file
-->         /breathing-space           app.Routes
+->         /                          app.Routes
```

Comment:
Does this mean that we won't have anymore the "breathing-space" context ?

Reply:
From the outside calling the API Platform it will still be "/individuals/breathing-space/endpoint-url" but internally (via curl micro-service) it will just be "/endpoint-url"

Reply:
I'm just worried about monitoring/logging if we have to search by URL in Graphana/Kibana for MTDP. How can these tools know for what service a specific URL is for? Maybe we won't any problem, but still not sure about this.